apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-settings/vo-settings-hierarchical-inclusion/vo-settings-hierarchical-inclusion.component.html

```diff
@@ -0,0 +1,33 @@
+<h1 class="page-subtitle">{{'VO_DETAIL.SETTINGS.HIERARCHICAL_INCLUSION.TITLE' | translate}}</h1>
+<div class="display-flex">
+  <perun-web-apps-refresh-button (click)="loadAllowedGroups()"></perun-web-apps-refresh-button>
+  <button (click)="addGroupsInclusion()" mat-flat-button class="mr-2 action-button" color="accent">
+    {{'VO_DETAIL.SETTINGS.HIERARCHICAL_INCLUSION.ADD' | translate}}
+  </button>
+  <button
+    [disabled]="selected.selected.length === 0"
+    (click)="removeGroupsInclusion()"
+    mat-flat-button
+    color="warn"
+    class="mr-2">
+    {{'VO_DETAIL.SETTINGS.HIERARCHICAL_INCLUSION.REMOVE' | translate}}
+  </button>
+  <div class="vo-search-select">
+    <perun-web-apps-vo-search-select
+      *ngIf="parentVos.length > 0"
+      [vos]="parentVos"
+      (voSelected)="voSelected($event)">
+    </perun-web-apps-vo-search-select>
+  </div>
+</div>
+<perun-web-apps-groups-list
+  *ngIf="!loading"
+  [tableId]="tableId"
+  [groups]="allowedGroups"
+  [selection]="selected"
+  [displayedColumns]="['select', 'id', 'name', 'description']"
+  [noGroupsAlert]="'VO_DETAIL.SETTINGS.HIERARCHICAL_INCLUSION.NO_GROUPS_ALLOWED_ALERT'"
+  (refreshTable)="loadAllowedGroups()"
```

Comment:
As of right now there is no way to trigger this refresh based on the displayed columns.

Reply:
`refreshTable` attribute removed from html